nucypher/blockchain/eth/agents.py

```diff
@@ -368,6 +368,8 @@ class StakingProviderInfo(NamedTuple):
         authorized: int
         operator_confirmed: bool
         index: int
+        deauthorizing: int
+        end_deauthorization: int
```

Comment:
@vzotova , @cygnusv should we be concerned that this may cause regression issues for existing 7.2.0 nodes when the contract is upgraded? Perhaps on node restart... 🤔

Reply:
Not an issue - node never calls `is_operator_confirmed`.